addons/binding/org.openhab.binding.chromecast/ESH-INF/i18n/chromecast_de.properties

```diff
@@ -0,0 +1,32 @@
+# binding
+binding.chromecast.name = Chromecast Binding
+binding.chromecast.description = Dieses Binding integriert Chromecast Ger�te (z.B. Chromecast, Chromecast Audio oder Chromecast Ultra).
+
+# thing types
+thing-type.chromecast.audiogroup.label = Chromecast Audiogruppe
+thing-type.chromecast.audiogroup.description = Audiogruppe von mehreren Chromecast Audio oder Media Playern.
```

Comment:
Or: "Audiogruppe mehrerer Chromecast Audio oder Media Player"
But I am not sure which sounds better.

Reply:
@CWempe Tahnks. I am gonna change it to: "Audiogruppe aus mehreren Chromecast Audio oder Media Playern". wdyt?